File: Semestrul 4/MPP/Proiecte/c#_gui/domain/Caz.cs
namespace csharp_avalonia.domain;

public class Caz : Entity<int>
{
    private string nume_caz;
    private string descriere;

    public Caz(int id, string nume_caz, string descriere)
    {
        this.id = id;
        this.nume_caz = nume_caz;
        this.descriere = descriere;
    }

    public Caz(string nume_caz, string descriere)
    {
        this.nume_caz = nume_caz;
        this.descriere = descriere;
    }
    
    
    public string Nume_caz { get => nume_caz; set => nume_caz = value; }
    
    public string Descriere { get => descriere; set => descriere = value; }

    public override string ToString()
    {
        return nume_caz;
    }
}

File: Semestrul 4/MPP/Proiecte/java_networking/Model/src/main/java/model/Caz.java
package model;

public class Caz extends Entity<Integer> {
    private String nume_caz;
    private String descriere;

    public Caz(int id, String nume_caz, String descriere) {
        this.id = id;
        this.nume_caz = nume_caz;
        this.descriere = descriere;
    }

    public Caz(String nume_caz, String descriere) {
        this.nume_caz = nume_caz;
        this.descriere = descriere;
    }

    public String getNume_caz() {
        return nume_caz;
    }

    public String getDescriere() {
        return descriere;
    }

    public void setNume_caz(String nume_caz) {
        this.nume_caz = nume_caz;
    }

    public void setDescriere(String descriere) {
        this.descriere = descriere;
    }

    @Override
    public String toString(){
        return nume_caz;
    }
}
